app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -148,12 +155,60 @@ class RefereeProfile extends Component {
       .catch(this.setErrorStateFromBackendData);
   };
 
-  hasPassedTest(level) {
-    const { referee: { certifications } } = this.state;
```

Comment:
You’re right, I should deactivate the “terminate statements with semicolon” setting in my formatting checker … 😄

Reply:
lol I was wondering about that